ui/src/layouts/storage/send_files_components.rs

```diff
@@ -1,28 +1,33 @@
-use std::path::PathBuf;
-
 use common::icons::outline::Shape as Icon;
 use common::{language::get_local_text_args_builder, MAX_FILES_PER_MESSAGE};
 use dioxus::prelude::*;
 use kit::elements::{button::Button, checkbox::Checkbox, Appearance};
-use uuid::Uuid;
+use warp::raygun::Location;
 
-use crate::layouts::storage::ChanCmd;
+use super::controller::StorageController;
 
 #[inline_props]
 pub fn file_checkbox(
     cx: Scope<'a>,
     file_path: String,
-    files_selected_to_send: UseRef<Vec<String>>,
+    storage_controller: UseRef<StorageController>,
     select_files_to_send_mode: UseState<bool>,
```

Comment:
I think `select_files_to_send_mode` would be better named as `is_selecting_files`.